c++/include/orc/Exceptions.hh

```diff
@@ -30,7 +30,7 @@ namespace orc {
   public:
     explicit NotImplementedYet(const std::string& what_arg);
     explicit NotImplementedYet(const char* what_arg);
-    virtual ~NotImplementedYet() ORC_NOEXCEPT;
+    virtual ~NotImplementedYet() ORC_NOEXCEPT override;
```

Comment:
Do we need to change this code inevitably because we don't have any option to disable warning errors?

Reply:
> Do we need to change this code inevitably because we don't have any option to disable warning errors?

Yes we can tell clang not warning this, but I think better to change this.

Reply:
Basically, we prefer to disable warnings in order to avoid code changes if possible, @zhanglistar .

Reply:
> Basically, we prefer to disable warnings in order to avoid code changes if possible, @zhanglistar .

Alright, I will rollback this.